dcat/index.html

```diff
@@ -3663,7 +3663,10 @@ <h4>Property: geometry</h4>
 <th>RDF Property:</th>
 <td><a data-cite="LOCN#locn:geometry"><code title="http://www.w3.org/ns/locn#geometry">locn:geometry</code></a></td>
 </tr>
+<!--
 <tr><th class="prop">Definition:</th><td>Associates any resource with the corresponding geometry. [[LOCN]]</td></tr>
+-->
+<tr><th class="prop">Definition:</th><td>Associates any spatial thing [[?SDW-BP]] with the corresponding geometry.</td></tr>
```

Comment:
is it 'a' rather than 'the' corresponding geometry?

Reply:
```suggestion
<tr><th class="prop">Definition:</th><td>Associates a spatial thing [[?SDW-BP]] with a corresponding geometry.</td></tr>
```